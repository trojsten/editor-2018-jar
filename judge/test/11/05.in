STR:
General
FGEPUNJ
STR:
Plukovnik
N
STR:
Podplukovnik
AA
STR:
Strateg
K
STR:
Spion
FUWU
STR:
Programator
MHTNNX
STR:
Velmajstermaxisupergeneralveduci
ORKEBRBS
STR:
Nadpodpredzadplukovnikazarovendokoncaajgeneral
WT
