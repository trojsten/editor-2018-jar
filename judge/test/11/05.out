{"Velmajstermaxisupergeneralveduci": "WT", "Spion": "MHTNNX", "Nadpodpredzadplukovnikazarovendokoncaajgeneral": "ORKEBRBS", "Programator": "N", "General": "K", "Strateg": "FUWU", "Podplukovnik": "FGEPUNJ", "Plukovnik": "AA"}